src/util/helpers.js

```diff
@@ -125,8 +125,8 @@ export const formatTime = date =>
     timeFormat(DATE_FORMAT_SPECIFIER)(new Date(date));
 
 export const getStartEndDateError = (startDate, endDate) => {
-    const start = parseTime(startDate);
-    const end = parseTime(endDate);
+    const start = parseTime(startDate.substring(0, 10)); // Only check date part
```

Comment:
What is the issue we're trying to solve here?  This check was introduced to catch invalid input like `01/12/yyyy`, so the time part shouldn't matter unless I'm missing something.  When is time input allowed?

Reply:
Keeping this after our discussions. The issue is that the backends adds the hour/minutes/seconds to periods - and I think we should keep this to be more future proof.